CarRental.Grpc.Producer/Grpc/Producer.cs

```diff
@@ -0,0 +1,82 @@
+﻿using CarRental.Grpc.Protos;
+using CarRental.Grpc.Server.Utils;
+using Grpc.Core;
+
+
+namespace CarRental.Grpc.Server.Grpc;
```

Comment:
Лишняя строка и не соответствие пути и неймспейса.
```suggestion

namespace CarRental.Grpc.Producer.Grpc;
```
Аналогично с другими классами проекта